contracts/protocol/clients/FermionFNFTPriceManager.sol

```diff
@@ -357,4 +380,58 @@ contract FermionFNFTPriceManager is FermionFractionsERC20Base, FermionErrors, IF
     function _isInvalidPriceError(bytes memory _reason) internal pure returns (bool) {
         return _reason.length == 4 && bytes4(_reason) == IPriceOracle.InvalidPrice.selector;
     }
+
+    /**
+     * @notice Adjusts the voter's records on transfer by removing votes if the remaining balance cannot support them.
+     *         This ensures the proposal's vote count remains accurate.
+     *
+     * @dev If the voter has no active votes or the current proposal is not active, no adjustments are made.
+     *      If the voter's remaining balance after the transfer is greater than or equal to their vote count,
+     *      no votes are removed.
+     *
+     * Reverts:
+     * - `OnlyCurrentERC20Clone` if the caller is not the current epoch's ERC20 clone contract.
+     *
+     * @param from The address of the sender whose votes may need adjustment.
+     * @param amount The number of fractions being transferred.
+     */
+    function adjustVotesOnTransfer(address from, uint256 amount) external {
+        FermionTypes.FermionFractionsStorage storage fractionStorage = Common._getFermionFractionsStorage();
+        uint256 currentEpoch = fractionStorage.currentEpoch;
+        address erc20Clone = fractionStorage.epochToClone[currentEpoch];
+
+        if (_msgSender() != erc20Clone) revert OnlyCurrentERC20Clone();
```

Comment:
When the epoch is changing, I suppose some ERC20 from the previous epoch remain in the user's balance (?).
If so, the users may want to transfer them, and this revert clause would deny that.
I think a `return` would be more appropriate than a `revert`